### PR TITLE
add kc edits about the package

### PR DESCRIPTION
**Pull request by KevCaz**
Hi @pedrohbraga, @linley-sherin and @katherinehebert 

This is a first draft  (English only) detailing the `qcbsRworkshops` features. I created a separate file `kc-edits.Rmd` for this that is not yet included in the bookdown.  IMHO, there are 2 ways to proceed: 

1- dispatching the content of `kc-edits.Rmd`  in the different chapters;
2- keeping it as a chapter on its own and add references to this new chapter in the other chapters. 

Let me know how do you want to proceed from here and also let me know if anything is missing or useless!

### CODE REVIEW

**Comment by linley-sherin** [comment]
Hi Kevin, sorry for the delayed response to this. What you have written looks good to me (waiting on other coordinators thoughts). I feel that this information is relevant to both developers and presenters so my vote is for integrating it as a separate chapter that we can refer to in the other sections - but I am open to hearing other peoples thoughts. I do have a question about whether this package will be impacted by the issue you brought up RE: Travis CI no longer being free come December 31st (if I understood that correctly).

File: kc_edits.Rmd [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+Finally, it can call `download_workshops()` to download the package:
+
+```R 
+# download workshop #1 + update CSS template + build both versions in HTML and pdf + extract the code and update template
+build_workshops(id =1, download = TRUE, pdf = TRUE, script = TRUE,  update_template = TRUE)
+```
+
+## Presenter email
+
+`mail_workshop()` writes the email to be sent tow weeks before the workshop, for instance:
```
Comment (linley-sherin) [inline kc_edits.Rmd:113]: Small spelling issue: 'two weeks'

File: kc_edits.Rmd [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+
+First it is often useful to clean up the (e.g., clean the cache), this is possible with `clean_workshops()`, for instance assuming workshop 1 was built in my current working directory, the cache can be cleaned like so: 
+
+```R
+clean_workshops(files = FALSE, cache = TRUE)
+```
+
+Second, it is possible to update the CSS template without rendering any presentation using `update_template()`. Similarly, it is possible to extract the R code without rendering a presentation using `extract_Rcode_workshop()`.
+
+Third, the first two slides (after the title slide) have now be standardized: it includes various hyperlinks (GitHub repository, wiki, PDF version, R script), and the list of package required (see for instance https://qcbsrworkshops.github.io/workshop01/workshop01-en/workshop01-en.html#2). Those tow slides can be generated using `first_slides()`:
```
Comment (linley-sherin) [inline kc_edits.Rmd:141]: Small spelling issue: 'two slides'

**Comment by KevCaz** [comment]
Thanks @linley-sherin, I'll take care of the two typos. 

> Travis CI no longer being free come December 31st (if I understood that correctly) - do you see this being an issue?

I read a bit about it (https://blog.travis-ci.com/2020-11-02-travis-ci-new-billing) and you are right, Travis CI will no longer be free (they'll give us 10k credits as a buffer). I wouldn't mind taking care of migrating from Travis CI to GitHub actions as I frequently use this CI service (the package is built with GH actions), and then replace `use_travis()` by `use_ghactions()`. Let me know what's best for you!

**Comment by linley-sherin** [comment]
Hello again @KevCaz , we have decided to discuss Travis CI at our next meeting (to be scheduled sometime this week) and then I can get back to you with how we want to proceed with that - thank you for offering to do the migration! I am thinking that I will wait until after the meeting to merge this pull request too just because once we merge we should edit the protocol for integration. I hope that is ok with you! I know it's been slow progress for us but I really appreciate the work you've been doing :)

**Comment by KevCaz** [comment]
Thanks @linley-sherin, I am working on a blog post to explain how to move from Travis to GH actions. Don't know when it'll be ready, but if someone wants to handle the migration, I guess the post will be helpful.

**Comment by KevCaz** [comment]
Here it s https://insileco.github.io/2020/11/24/continuous-integration-for-r-projects-from-travis-ci-to-github-actions-step-by-step/.

**Pull request merged**